app/models/global_settings.rb

```diff
@@ -1,27 +1,24 @@
 class GlobalSettings < ActiveRecord::Base
 
-  attr_accessible :command_apache_restart, :command_apache_vhosts, :command_apache_status, :sites_enabled_path, :smtp_timeout, :asynchronous, :bing_api, :beef_url, :reports_refresh
+  attr_accessible :smtp_timeout, :asynchronous, :bing_api, :beef_url, :reports_refresh, :singleton
 
-  validates :command_apache_restart, :presence => true, :length => {:maximum => 255}
-  validates :command_apache_vhosts, :presence => true, :length => {:maximum => 255}
-  validates :sites_enabled_path, :presence => true, :length => {:maximum => 255}
   validates :smtp_timeout, :presence => true, :length => {:maximum => 2},
             :numericality => {:greater_than_or_equal_to => 1, :less_than_or_equal_to => 20}
 
-  def self.asynchronous?
-    first.asynchronous?
-  end
+  validates_inclusion_of :singleton, in: [0]
 
-  def self.apache_status
-    `#{first.command_apache_status} 2>&1`
+  def self.instance
+    begin
+      find(1)
```

Comment:
Is this because it is lazy-populated? E.g. the first time you tried it it didn't exist. When you changed it the record had been created?